monarch_ingest/string/protein_links.py

```diff
@@ -1,18 +1,28 @@
 import re
 import uuid
+import logging
 
 from biolink_model_pydantic.model import Gene, PairwiseGeneToGeneInteraction, Predicate
 from koza.cli_runner import koza_app
 
+logger = logging.getLogger(__name__)
+logger.setLevel("DEBUG")
+
 row = koza_app.get_row()
 entrez_2_string = koza_app.get_map('entrez_2_string')
 
 pid_a = row['protein1']
 gene_ids_a = entrez_2_string[pid_a]['entrez']
+if not gene_ids_a:
+    logger.error(f"protein1 PID '{str(pid_a)}' has no Entrez mappings?")
```

Comment:
I'd say let's set these map messages to debug, but also not force the level of debug - we have a lot of noisy logs that we're not actually doing anything with right now

Reply:
ok